Build Project Issue

Error Build after upgraded Android Studio to the latest version.

Error:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':wear_plus:compileDebugKotlin'.
> Inconsistent JVM-target compatibility detected for tasks 'compileDebugJavaWithJavac' (1.8) and 'compileDebugKotlin' (21).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+# 1.2.2
+
+- Fixes build issue
+
 # 1.2.1
 
 - Updates proguard rules for R8

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -38,6 +38,13 @@ android {
     lintOptions {
         disable 'InvalidPackage'
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
+    }
+    kotlinOptions {
+        jvmTarget = '11'
+    }
     namespace "dev.rexios.wear_plus"
 }
 

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: wear_plus
 description: An actively maintained plugin that offers Flutter support for Wear OS by Google
-version: 1.2.1
+version: 1.2.2
 homepage: https://github.com/Rexios80/wear_plus
 
 environment:

